genrl/core/actor_critic.py

```diff
@@ -234,3 +251,19 @@ def get_actor_critic_from_name(name_: str):
     if name_ in actor_critic_registry:
         return actor_critic_registry[name_]
     raise NotImplementedError
+
+
+class MlpCentralizedActorCritic(BaseActorCritic):
+    def __init__(
+        self, state_dim: int, action_dim: int, shared_layers: Tuple = (512, 256)
+    ):
+        super(CentralizedActorCritic, self).__init__()
+
+        self.state_dim = state_dim
+        self.action_dim = action_dim
+
+        self.shared = mlp([self.state_dim] + list(shared_layers))
+        self.actor = nn.Linear(shared_layers[-1], self.action_dim)
+        self.critic = nn.Linear(shared_layers[-1], 1)
+
+        init_weights(self.parameters())
```

Comment:
I'd say keep this between `MlpSingleActorMultiCritic` and `CNNActorCritic`. Also add it to the `actor_critic_registry`